integration/simulation/network/network.go

```diff
@@ -31,9 +33,10 @@ type RPCHandles struct {
 	// an obscuro client per obscuro node in the network (used for things like validation rather than transactions on behalf of sim accounts)
 	ObscuroClients []rpcclientlib.Client
 
-	// an RPC client per wallet, with a viewing key setup (on the client and registered on its corresponding host enclave),
+	// an RPC client per node per wallet, with a viewing key setup (on the client and registered on its corresponding host enclave),
```

Comment:
@BedrockSquirrel Can we add a TODO here that in the future, we can hopefully drop back to a single RPC client per node?